test/functional/apps/discover/group1/_no_data.ts

```diff
@@ -55,16 +61,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       const button = await testSubjects.find('createDataViewButton');
       button.click();
+      log.error('Create dataview button clicked');
       await retry.waitForWithTimeout('data view editor form to be visible', 15000, async () => {
         return await (await find.byClassName('indexPatternEditor__form')).isDisplayed();
       });
 
+      log.error('indexPatternEditor__form is displayed');
       const dataViewToCreate = 'logstash';
       await createDataView(dataViewToCreate);
+      log.error('data view created');
       await PageObjects.header.waitUntilLoadingHasFinished();
+      log.error('loading finished');
+      // await browser.refresh();
```

Comment:
@majagrubic @clintandrewhall so my suggested change, evaluated in this quick PR to checked fixed the problem, so a few more cleanups and we should be good to merge 🎉